openpype/plugins/publish/collect_audio.py

```diff
@@ -0,0 +1,83 @@
+import pyblish.api
+from pprint import pformat
+
+from openpype.client import (
+    get_last_version_by_subset_name,
+    get_representations,
+)
+from openpype.pipeline import (
+    legacy_io,
+    get_representation_path,
+)
+
+
+class CollectAudio(pyblish.api.InstancePlugin):
+    """ Collecting available audio subset to instance
+
+    """
+    label = "Collect Audio"
+    order = pyblish.api.CollectorOrder + 0.1
+    families = ["review"]
+    hosts = [
+        "nuke",
+        "maya",
+        "shell",
+        "hiero",
+        "premiere",
+        "harmony",
+        "traypublisher",
+        "standalonepublisher",
+        "fusion",
+        "tvpaint",
+        "resolve",
+        "webpublisher",
+        "aftereffects",
+        "flame",
+        "unreal"
+    ]
+
+    audio_subset_name = "audioMain"
+
+    def process(self, instance):
+        if instance.data.get("audio"):
+            self.log.info(
+                "Skipping Audio collecion. It is already collected"
+            )
+            return
+
+        # Add audio to instance if exists.
+        self.log.info('Collecting Audio Data ...')
```

Comment:
I feel this would be more useful if it would state what audio subset name it would look for.
```suggestion
        self.log.info("Searching for audio subset '{subset}' in asset '{asset}'".format(asset=instance.data["asset"], self.audio_subset_name))
```
☝️ Note: Line length of course is too long in suggested example. Consider psuedocode